### PR TITLE
#184948260 Delete seller product

### PR DESCRIPTION
**Pull request by khevin2**
### What does this PR do?
This PR add allows a `Merchant/vendor` to delete a product in his own collection
### Description of Task to be completed?
- Add a button to delete a product in products list
- Add confirm Modal before deleting
- Delete a product on the backend
### How should this be manually tested?
- Clone this repo `git clone https://github.com/atlp-rwanda/e-comm-team-amigos-fn`
- Switch your working directory `cd e-comm-team-amigos-fn`
- Install all the dependencies by running `npm install`
- Then run the project by typing `npm start`
### Any background context you want to provide?
- Have Node Js installed in your local machine
- Have latest web browser
### What are the relevant pivotal tracker?
Story ID:  [#184948260](https://www.pivotaltracker.com/story/show/184948260)
### Screenshots

image

image

image

image

### CODE REVIEW

File: src/redux/actions/sellerCollection.actions.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+});
+
+export const deleteProductFailure = () => ({
+	type: DELETE_PRODUCT_ERROR,
+});
+
+export default function authHeader() {
+	//   const token = JSON.parse(sessionStorage.getItem("token"));
+	const token =
+		'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiJmMmMzMDA3MC04MjQxLTQzYWEtYWY0YS04YmQwM2YzZDVkNjMiLCJ1c2VyRW1haWwiOiJjeXVzYS5raGV2ZW5Ab3V0bG9vay5jb20iLCJpYXQiOjE2ODM0NjEzNTEsImV4cCI6MTY4MzQ2NDk1MX0.EiceFpFMpTou5-W_XPkhnV2FOS35bZIarAtTugGHgDc';
```
Comment (Emile-SHUMBUSHO) [inline src/redux/actions/sellerCollection.actions.js]: Don't forget to replace this static token with dynamic one.

**Comment by Emile-SHUMBUSHO** [comment]
Provide a screenshots for your UI implemented design

**Review by GasanaJr — COMMENTED** [review]
add some unit tests

**Review by GasanaJr — APPROVED** [review]
LGTM

**Review by GasanaJr — CHANGES_REQUESTED** [review]
I am facing some issues, I am deleting a product but it re-renders the product again. Look into what might be causing the behaviour

**Review by wilbrord2 — APPROVED** [review]

**Review by wilbrord2 — APPROVED** [review]

**Review by GasanaJr — CHANGES_REQUESTED** [review]
Rebase and resolve conflicts

**Review by AbdulKhaliq59 — APPROVED** [review]

**Review by GasanaJr — APPROVED** [review]

**Review by Mystica52 — APPROVED** [review]

**Review by Mystica52 — APPROVED** [review]
working correct unless rebase

**Comment by Emile-SHUMBUSHO** [comment]
This PR has a conflict and won't be merged unless resolved.

**Review by Habinezajanvier — APPROVED** [review]
👍

**Pull request merged**